omero/sysadmins/index.rst

```diff
@@ -43,9 +43,51 @@ series of commands.
     unix/server-install-homebrew
     unix/server-binary-repository
     unix/server-postgresql
-    unix/install-web
```

Comment:
Perhaps further re-order:

 * "General background"
 * Server installation
 * Optimizing ...
 * Web installation
 * Web maintenance and customization

import/storage should perhaps go up to general background.

Reply:
I think I'd favour just making this new web stuff 1 subsection.

We might have anchor link issues if we start moving other things around too - I definitely remember using them somewhere.

Reply:
I have re-organized a bit the page but not pushed yet 
i have not noticed any anchor issues so far

Reply:
The anchors might be on the website

Reply:
Do you have any idea which ones?

Reply:
I think the 'Optimizing OMERO as a data repository' and maybe the import section. I'll check now but we haven't actually removed either of these currently so we're ok anyway - it was just that Josh was suggesting doing so. Although maybe if we're having all the customization stuff in this new web section, it isn't worth keeping the data repo one - that stuff is much better highlighted on the new website now.

Reply:
Okay - having grepped - there's a link to the import section anchor in the 5.0.1 release announcement and a link to the data repo anchor on http://www.openmicroscopy.org/omero/publish/

Reply:
Discussed with @hflynn re-organising the index page
and adjusting headers